### PR TITLE
Update AjaxUploader.jsx

### PR DESCRIPTION
**Pull request by thegatheringstorm**
Change 'for' structure to 'map' structure to make it more readable.

### CODE REVIEW

File: src/AjaxUploader.jsx [hunk 1]
```diff
@@ -75,12 +75,10 @@ class AjaxUploader extends Component {
 
   uploadFiles(files) {
     const postFiles = Array.prototype.slice.call(files);
-    const len = postFiles.length;
-    for (let i = 0; i < len; i++) {
-      const file = postFiles[i];
+    postFiles.map((file) => {
```
Comment (afc163) [inline src/AjaxUploader.jsx]: `forEach` is much better.

**Comment by afc163** [comment]
The Travis CI build failed.

File: src/AjaxUploader.jsx (hunk 1 above)
Comment (thegatheringstorm) [reply src/AjaxUploader.jsx]: I'm sorry to use 'map' structure.Yes,'forEach' is much better..

**Comment by thegatheringstorm** [comment]
@afc163  How about this.I'm looking forward to your suggestion.
Thanks. :smile:

**Comment by benjycui** [comment]
rc-upload@2.4.2

**Comment by thegatheringstorm** [comment]
Thanks!

**Pull request merged**